.github/labeler.yml

```diff
@@ -0,0 +1,48 @@
+GHA:
+  - .github/**/*
+
+config:
+  - charts/registry/config/**/*
+
+docs:
+  - README.md
+  - docs/**/*.md
+
+build:
+  - Makefile
+  - Dockerfile
+  - docker-bake.hcl
+  - dockerfiles/**/*
+
+ci:
+  - .github/**/*
+  - tests
+  - testutil
+
+vendor:
+  - vendor/**/*
+
+storage:
+  - registry/storage
+
+s3:
+  - registry/storage/s3-aws
+
+gcs:
+  - registry/storage/gcs
+
+azure:
```

Comment:
Is this the label that is applied? (so if `registry/storage/azure` is touched, we apply `azure` as label?)

Because we have existing `area/xxx` labels; I think it would make sense to use those (same for various other ones);

<img width="313" alt="Screenshot 2023-12-20 at 17 15 01" src="https://github.com/distribution/distribution/assets/1804568/9372bffd-bfc1-4123-92b4-e5ebcec14790">

Reply:
We have the manually created ones by us. Labeler action will create new ones but it's OK IMHO. We can always clean up the old cruft.

Reply:
I like the `area/xx` prefixes, as they are clear on what "dimension" the label is for. 😅

Reply:
Ok will change, sir 😄

Reply:
Ok I've updated them all. Now the labeler will use all the existing labels and they have `area` prefix 😄